File: OnlinePK-Android/biz-live/src/main/java/com/netease/biz_live/yunxin/live/model/LiveConfig.java
/*
 * // Copyright (c) 2021 NetEase, Inc.  All rights reserved.
 * // Use of this source code is governed by a MIT license that can be found in the LICENSE file.
 */

package com.netease.biz_live.yunxin.live.model;

import java.io.Serializable;

/**
 * 直播参数
 */
public class LiveConfig implements Serializable {
    public String httpPullUrl;//String	直播拉流地址
    public String rtmpPullUrl;//String rtmp直播拉流地址
    public String hlsPullUrl;//Stringhls拉流地址
    public String pushUrl;//String	推流地址pushUrl
    public String cid;// String	直播频道Cid
}

File: OnlinePK-Android/lib-live-room-service/src/main/java/com/netease/yunxin/lib_live_room_service/bean/LiveConfig.kt
/*
 *  Copyright (c) 2021 NetEase, Inc.  All rights reserved.
 *  Use of this source code is governed by a MIT license that can be found in the LICENSE file
 */

package com.netease.yunxin.lib_live_room_service.bean

import java.io.Serializable

/**
 * 直播参数
 */
data class LiveConfig(//	直播拉流地址
    val httpPullUrl: String,
    // rtmp直播拉流地址
    val rtmpPullUrl: String,
    //hls拉流地址
    val hlsPullUrl: String,
    //	推流地址pushUrl
    val pushUrl: String,
    //	直播频道Cid
    val cid: String
) : Serializable